src/components/Hero.tsx

```diff
@@ -1,75 +1,128 @@
-                                onClick={() => window.open(soc.link, "_blank")}
-                                className="w-10 h-10 flex items-center cursor-pointer"
-                            >
-                                <Image
-                                    src={soc.icon}
-                                    alt="source code"
-                                    width={100}
-                                    height={100}
-                                    className="w-4/5 h-4/5 object-contain"
-                                />
-                            </div>
-                        ))}
-                    </div>
-                </div>
-                {/* redirects to /auth for spotify authentication */}
-                <button
-                    className="inline-block rounded border-2 border-info px-6 mt-4 pb-[6px] pt-2 text-xs font-medium uppercase leading-normal text-info transition duration-150 ease-in-out hover:border-info-600 hover:bg-info-50/50 hover:text-info-600 focus:border-info-600 focus:bg-info-50/50 focus:text-info-600 focus:outline-none focus:ring-0 active:border-info-700 active:text-info-700 motion-reduce:transition-none dark:hover:bg-cyan-950 dark:focus:bg-cyan-950"
-                    onClick={auth}
-                >
-                    Authorize
-                </button>
+        <div className="flex flex-col justify-center text-center items-center mt-4 animate-fade-up animate-delay-[2400ms] animate-ease-in">
+          <p className="pr-1 text-lg text-balance">Created by Calvin Yu, Check out my portfolio and socials!</p>
+          <div className="flex">
+            {socials.map((soc, index) => (
+              <div
+                key={index}
+                onClick={() => window.open(soc.link, "_blank")}
+                className="w-10 h-10 flex items-center cursor-pointer"
+              >
+                <Image
+                  src={soc.icon}
+                  alt="source code"
+                  width={100}
+                  height={100}
+                  className="w-4/5 h-4/5 object-contain"
+                />
+              </div>
+            ))}
+          </div>
+          {/* redirects to /auth for spotify authentication */}
+          <button
+            className="inline-block rounded border-2 border-info px-6 mt-2 pb-[6px] pt-2 text-xs font-medium uppercase leading-normal text-info transition duration-150 ease-in-out hover:border-info-600 hover:bg-info-50/50 hover:text-info-600 focus:border-info-600 focus:bg-info-50/50 focus:text-info-600 focus:outline-none focus:ring-0 active:border-info-700 active:text-info-700 motion-reduce:transition-none hover:bg-cyan-950 focus:bg-cyan-950 w-3/4 sm:w-2/4"
+            onClick={auth}
+          >
+            Authorize
+          </button>
+          {/* disclaimer */}
+          <div
+            className={`transition-opacity duration-500 ease-in-out opacity-${
+              isVisible ? "100" : "0"
```

Comment:
nit: this should be named more descriptively like `isDisclaimerVisible`. You would use `isVisible` if it applies to the entire component rather than a portion of it.